src/devices/README.md

```diff
@@ -17,7 +17,7 @@ We may publish NuGet packages of device bindings at a later date.
 
 ## Contributing a binding
 
-Anyone can contribute a binding. Please do! Bindings should follow the model that is used for the [Mcp23xxx](Mcp23xxx/README.md) or [Mcp3008](Mcp3008/README.md) implementations.  There is a [Binding Markdown template](Device%20Binding%20Template.md) that can help you get started, as well.
+Anyone can contribute a binding. Please do! Bindings should follow the model that is used for the [Mcp23xxx](Mcp23xxx/README.md) or [Mcp3008](Mcp3008/README.md) implementations.  There is a [Template Binding Template](../../tools/templates/DeviceBindingTemplate/README.md) that can help you get started, as well.
```

Comment:
> Template Binding Template

did you mean Device Binding Template?

Reply:
LOL.  Yup, I made a few changes trying to get the link correct and must have wrote wording wrong on last go.  Thx.  Updated.